eslint.config.js

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+
+import eslint from "@eslint/js";
+import tseslint from "typescript-eslint";
+import eslintConfigPrettier from "eslint-config-prettier";
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.strict,
+  eslintConfigPrettier,
```

Comment:
Is there a reason this isn't spread? 
`...eslintConfigPrettier`

Reply:
`eslintConfigPrettier` is not an array. It just makes sure that any eslint rules that would conflict with prettier settings are turned off, so that eslint and prettier play nice. The gitHub docs for this package do not spread it either. 

`import someConfig from "some-other-config-you-use";
import eslintConfigPrettier from "eslint-config-prettier";

export default [
  someConfig,
  eslintConfigPrettier,
  {
    rules: {
      indent: "error",
    },
  },
];`

Reply:
cool
separately, you can use ``` for multi line code stuff

```
Look at this multiline goodness
function look(){
  return 'sweet'
}